src/mesh/mod.rs

```diff
@@ -173,6 +189,40 @@ impl<Sh: Shape + Clone> ShapeMesh<Sh> {
 
         Ok(())
     }
+
+    // gibt den Punkt am Rand der Form zurück in der Richtung, in die wir gucken
```

Comment:
Doc comments need `///`. Also: German. What about:

```rust
/// Returns the point on the shape's surface the camera is currently looking at.
```

We might want to generalize this ray marching later on, but for now it's OK to keep here.